commands/stats.js

```diff
@@ -88,22 +88,67 @@ const stats = msg => {
 
 	const getCompUserStats = (error, response, parsedBody) => {
 		const competitiveData = parsedBody.us.stats.competitive;
+		const compGmStats = competitiveData.game_stats;
+		const compOvrStats = competitiveData.overall_stats;
 		const emptyObjectCompChecker = Object.getOwnPropertyNames(competitiveData);
 		if (emptyObjectCompChecker.length === 0 || emptyObjectCompChecker.length === 0) {
 			msg.reply("Sorry. This user has missing data. Boo boo doo de doo.");
 			return;
 		}
-		// Sucrizzle - Playing with Git commits
+
+		// Make sure the user has played some games this comp Season.  Not sure if it would be null or 0 or ???  Taking a guess here
+		if (compOvrStats.games == null || compOvrStats.games == 0) {
```

Comment:
I'm going to merge this code but I want to change these variable names a bit, because I think it's slightly confusing. compOvrStats would be better as compOverallStats. Just personal taste though because I got confused. I can do it in a future update though.